src/utilits/randomNumberProduct.js

```diff
@@ -0,0 +1,8 @@
+export function getRandomNumbers() { 
```

Comment:
```suggestion
export function getRandomNumbers(max, min = 0) { 
```
Its better to make this code reusable + flexible.